elpis/engines/kaldi/objects/model.py

```diff
@@ -133,20 +133,21 @@ def prepare_for_training():
 
             with path_file_path.open(mode='w') as fout:
                 with path_resource.open() as fin:
-                    content = pystache.render(
-                        fin.read(),
+                    template = Template(fin.read())
+                    content = template.render(
```

Comment:
If these template variables aren't being used again, can you just `Template(fin.read()).render(...`?

Reply:
Fair point...